Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -270,4 +278,45 @@ protected function getFixersFromConfig($fileName, $config)
 
         return $fixers;
     }
+
+    /**
+     * Parses the '--REQUIREMENTS--' block of a '.test' file and determines requirements.
+     *
+     * @param string $fileName
+     * @param string $config
+     *
+     * @return array
+     */
+    protected function getRequirementsFromConfig($fileName, $config)
+    {
+        $requirements = array('hhvm' => true, 'php' => PHP_VERSION);
```

Comment:
php key requires to be at least with given version.
and... hhvm requires to run under hhvm? 

the documentation for possible requirements is needed